src/encoded/static/components/matrix_reference_epigenome.js

```diff
@@ -246,10 +239,14 @@ const convertReferenceEpigenomeToDataTable = (context, expandedRowCategories, ex
                             const colIndex = colMap[colMapKey].col;
                             cells[colIndex] = {
                                 content: (
-                                    <a href={`${context.search_base}&${rowCategoryQuery}&${subCategoryQuery}&${colMap[colMapKey].query}`} style={{ backgroundColor: rowSubcategoryColor }}>
-                                        <span className="sr-only">Search {rowCategoryBucket.key}, {rowSubcategoryBucket.key} for {rowSubcategoryColCategoryBucket.key}, {cellData.key}</span>
-                                    </a>
+                                    <React.Fragment>
```

Comment:
Not important- A newer alternative to React.Fragment is to use "<>". See https://reactjs.org/docs/fragments.html#short-syntax

Reply:
I don’t like how `<>` looks so I’ve decided not to use it myself. Nothing wrong with others using it.